### PR TITLE
[releasing] Add more emphasis on the importance of ensuring that the release lands on time.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
This PR requires approval from all team members.

@randallli  randallli
 @ianegordon ianegordon
 @romoore romoore
 @galiak11 galiak11
 @yarneo yarneo
 @codeman7 codeman7
 @andrewoverton andrewoverton
 @wenyuzhang666 wenyuzhang666

**Review by codeman7 — APPROVED** [review]

**Review by yarneo — APPROVED** [review]

File: contributing/releasing.md
Comment (romoore) [inline contributing/releasing.md]: How are work hours defined for us?  11am - 3pm NY Time?

File: contributing/releasing.md
Comment (romoore) [inline contributing/releasing.md:27]: Do we have steps on how to "hand off a release"? A checklist or something maybe?

**Review by romoore — APPROVED** [review]

**Comment by jverkoey** [comment]
Waiting for LGTM from:

@randallli randallli
 @ianegordon ianegordon
 @galiak11 galiak11
 @andrewoverton andrewoverton
 @wenyuzhang666 wenyuzhang666

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md:27]: Filed https://github.com/material-components/material-components-ios/issues/4980 to track writing this guide.

Comment (jverkoey) [reply contributing/releasing.md]: Made the wording here more clear by adding 3PM as the cut-off point. Roll-backs tend to happen between 1-4 hours after a CL lands so this would result in the team being forced to stay after-hours to handle the fall-out.

File: contributing/releasing.md
Comment (yarneo) [inline contributing/releasing.md:30]: If you are blocking it after 3pm then the likelihood of a release starting and going out on Wedensday is low.

**Review by yarneo — CHANGES_REQUESTED** [review]

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md:30]: Moving discussion to the thread above.

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md]: @yarneo 
> If you are blocking it after 3pm then the likelihood of a release starting and going out on Wedensday is low.

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md]: Made the wording here more clear by adding 3PM as the cut-off point. Roll-backs tend to happen between 1-4 hours after a CL lands so this would result in the team being forced to work after-hours to handle the fall-out.

Comment (jverkoey) [reply contributing/releasing.md]: > If you are blocking it after 3pm then the likelihood of a release starting and going out on Wedensday is low.
> — @yarneo

Comment (jverkoey) [reply contributing/releasing.md]: What should we do here?

File: contributing/releasing.md
Comment (jverkoey) [reply contributing/releasing.md:30]: GitHub keeps hiding the other thread because it's "outdated" :(

What should we do here? Agreed that 3pm can be tough for a release to land by, though it should be doable.

File: contributing/releasing.md
Comment (romoore) [reply contributing/releasing.md]: ~@jverkoey Do you mean "the release starting and going out on **Thursday** is low"?~

Oops, I now see that was meant as a quote.  I think "clean" releases tend to take a few hours with most of the time spent Tuesday morning doing the internal testing and writing the descriptions. These are parallel operations.

File: contributing/releasing.md
Comment (yarneo) [reply contributing/releasing.md:30]: it's doable if all the testing pass fast, there are not many flakes, and you get an immediate thumbs up from all owners that broke. Otherwise I see it being extremely difficult. As you can see most of these time spenders aren't in our control. If an average releaser gets in at 10:30 and has lunch then he is given basically 4 hours from start to finish.

There are a few options I see:
1. We start releases Tuesday and pull the trigger Wednesday morning.
2. We make it OK to have releases land Thursday as well.
3. We remove the workday criteria and have the releaser release it when he sees fit to achieve the given deadline.

Comment (romoore) [reply contributing/releasing.md:30]: What is "pulling the trigger" in the release process? Starting testing?

I think it's always been OK for releases to land on Thursdays. Did we change this recently?

Comment (yarneo) [reply contributing/releasing.md:30]: pulling the trigger = submitting CL (after making sure the day before everything is good), publishing release on GitHub and CocoaPods.

**Review by galiak11 — APPROVED** [review]

**Review by ajsecord — APPROVED** [review]

**Comment by jverkoey** [comment]
Am dropping this PR. I'm not sure there's too much value in giving additional guidance here.

**Pull request closed without merging**
